frb/data/Galaxies/180924/FRB180924_host.json

```diff
@@ -71,10 +80,6 @@
         "DES_z": 19.832713667943604,
         "DES_z_err": 0.020038900000000002,
         "EBV": 0.0166,
-        "VLT_FORS2_I": 20.071526286300077,
```

Comment:
These should not have been removed.  
You should re-run 180924 and make sure
they reappear.

Reply:
Done, they now appear.